lib/dnsimple.js

```diff
@@ -22,7 +22,7 @@ const services = {
 Dnsimple.services = services;
 Dnsimple.DEFAULT_TIMEOUT = require('http').createServer().timeout;
 Dnsimple.DEFAULT_BASE_URL = 'https://api.dnsimple.com';
-Dnsimple.DEFAULT_USER_AGENT = 'dnsimple-node/v2';
+Dnsimple.DEFAULT_USER_AGENT = 'dnsimple-node/2.4.0';
```

Comment:
I suggest to extract it in a value called Version, somewhere, as we do for all the other clients.